npm/build-docs.js

```diff
@@ -4,6 +4,7 @@
 // ---------------------------------------------------------------------------------------------------------------------
 
 /* eslint-env node, es6 */
+/* eslint no-console: off */
```

Comment:
All these rules can be placed in `npm/.eslintrc` instead of repeating them across all files.